docs/01-readme.md

```diff
@@ -49,6 +51,8 @@ sudo apt build-dep awesome
 git clone https://github.com/awesomewm/awesome
 cd awesome
 make package
+cd build
+make distclean package CMAKE_ARGS="-DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=None -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_INSTALL_LOCALSTATEDIR=/var -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_INSTALL_LIBDIR=lib/x86_64-linux-gnu -DSYSCONFDIR=/etc"
```

Comment:
This does not work. I tried following the instructions (more or less, I didn't clone from GitHub but from my local git repo, but that shouldn't make a difference, I guess).
```
$ LC_ALL=C make distclean
make: *** No rule to make target 'distclean'.  Stop.
```
However, half of this isn't necessary, I think. Just add the `CMAKE_ARGS` directly to the first `make package` invocation. The `make distclean` isn't needed when one freshly cloned the repo.

For the CMake arguments: Well, this seems to be what Debian uses on x86-64. I'm not sure how much of this is really necessary for users. I would guess that the following would be enough, but cannot really test:
```
make package CMAKE_ARGS="-DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_SYSCONFDIR=/etc -DSYSCONFDIR=/etc"
```

Reply:
okay, so now i edit the pull request? or do i submit a new one with the edit that you outlined?